code/src/index.css

```diff
@@ -1,13 +1,95 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+form {
+  display: flex;
+  flex-direction: column;
 }
+
+.container {
+  margin: 100px auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 400px;
+  width: 100%;
+}
+
+.new-comment {
+  width: 100%;
+  background-color: rgb(239 239 239);
+  border: 2px solid gray;
+  padding: 10px;
+  margin-bottom: 30px;
+  box-shadow: 7px 7px black;
+  font-size: small;
+  display: flex;
+  flex-direction: column;
+}
+
+.new-comment__text {
+  height: 55px;
+  margin-top: 10px;
+}
+
+.new-comment__error {
+  color: red;
+  font-style: italic;
+}
+
+.new-comment__characters {
+  color: black;
+}
+
+.new-comment__characters--color-red {
+  color: red;
+}
+
+.new-comment__button {
+  width: 210px;
+  border-radius: 30px;
+  height: 35px;
+  margin-top: 10px;
+  background-color: #ffa3ac;
+  border: none;
+}
+
+.preview-comment {
+  width: 100%;
+  border: 2px solid gray;
+  padding: 10px;
+  margin-bottom: 30px;
+  box-shadow: 7px 7px black;
+}
+
+.preview-comment__details {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.preview-comment__button {
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  border: none;
+}
+
+.preview-comment__button--color-rose {
+  background-color: #f5afaf;
+}
+
+.preview-comment__heart, .preview-comment__time {
+  color: gray;
+}
+
+@media (max-width: 666px) {
```

Comment:
not sure here but, is there a special reason you use 666px? 
And not 768 for example which is the tablet view ? :)